package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "todolist",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open",
```

Comment:
- [ ] When I try to start your app with `npm start` I get this error. Kindly ensure your install webpack server as suggested by the message.

  ![web-se](https://user-images.githubusercontent.com/54848363/184946336-b27747e6-d2ac-4eed-ab06-af4ea88c3bd1.jpg)